### DISCUSSION TITLE
stock portfolio, decimals remove

### DISCUSSION THREAD

**Discussion by DevAnyKR** [moneymanagerex/moneymanagerex#6604] [Q&A]
![image](https://github.com/moneymanagerex/moneymanagerex/assets/110871727/8f0cd216-70a4-4c43-8118-ec3dedafa7b5)
![image](https://github.com/moneymanagerex/moneymanagerex/assets/110871727/550a93d5-c6e1-460e-928f-2d47f4c27b1d)

![image](https://github.com/moneymanagerex/moneymanagerex/assets/110871727/34f47d46-aea0-4e5d-b34f-768e1ecb63c0)

There are fields that show decimal points. I want to remove them.

**Comment by nickallington** [comment]
That could be a set-up option, but decimals are required for prices in most other currencies so the fields can't be Integer.

**Reply by DevAnyKR** [reply]
In Korea, we do not use decimal points. No decimal point required.

**Reply by nickallington** [reply]
Perhaps the currency table could also include a decimals setting for each currency, but that would be for the tech community to decide. It might be harder than it sounds!

**Reply by n-stein** [reply]
The CURRENCYFORMATS_V1 table already stores the decimal for each currency in the SCALE field, and there is a method that formats values according to the given currency. Seems we are not applying it here at the moment.

**Comment by nickallington** [comment]
Understood, KR is not the only country with no decimals. My point was only that the field cannot be Integer as other countries do have decimals, but it could be or become a user-definable option.

**Reply by DevAnyKR** [reply]
Can't change the number of decimal places to 0 or 1 in settings.
![image](https://github.com/user-attachments/assets/96f7f439-986a-4e9b-a037-8a339b33ef12)

Settings do not apply to Grid.
![image](https://github.com/user-attachments/assets/1411f92a-86a0-4b56-bdbb-af87bd824618)

That's why I posted this.

This post should now be turned into an issue.
